File: lab/python/textos/contar_texto.py
def retornar_informacoes_arquivo(arquivo):
    with open(arquivo, "r") as file:
        texto = file.read()
        print(f"caracteres: {len(texto)}")
        print(f"linhas: {len(texto.splitlines())}")
        print(f"palavras: {len(texto.split())}")


retornar_informacoes_arquivo("requirements.txt")

File: lab/javascript/textos/contar_texto.js
const fs = require("fs");

function retornarInformacoesArquivo(arquivo) {
    try {
        let texto = fs.readFileSync(arquivo, "utf-8");

        console.log(`caracteres: ${texto.length}`);
        console.log(`linhas: ${texto.split(/\r?\n/).length}`);
        console.log(`palavras: ${texto.split(/\s+/).filter(Boolean).length}`);
    } catch (err) {
        console.error("Erro ao ler o arquivo:", err.message);
    }
}

retornarInformacoesArquivo("/home/debianjv/Documentos/estudos_logica/requirements.txt");
